hack/tools/go.mod

```diff
@@ -0,0 +1,5 @@
+module tools
+
+go 1.12
+
+require github.com/corneliusweig/release-notes v0.0.0-20191014214505-0be5c7c66752
```

Comment:
maybe tag a release?

Reply:
let's say I have this

v0.3.1-alpha.1 -->  v0.3.1 --> v0.3.2 --> v0.3.3-alpha.1 --> v0.3.3

when I generate notes for v0.3.3, does it compute since v0.3.2 –or since v0.3.3-alpha.1?

Reply:
That would only include in the range v0.3.3-alpha.1 to present. But it totally makes sense to change that :)

Reply:
Yeah we’d only want changes between revisions went to the user.

Reply:
I've introduced an option for the tool, so that it now calculates the difference since v0.3.2 and skips the alpha version. Also, I tagged a release :)